notebooks/pymc4_design_guide.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Whats a value property? In this particular case it would be really helpful to show those three concepts are they relate to pymc3, perhaps not in a code block but something that we can refer to in the pymc3 docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yeah actually that's not very correct, rephrased.
FWIW, as long as random variable is tensor/array-like (an object with operator overloaded so you can add, multiple, etc) and with a log_prob method you can evaluate it on itself (which is what I meant by its underlying value here).

Reply:
Sounds good, 
The one last thing that's not inference specific but is a nice PyMCX design feature is the object also act as independent distributions ("Free context" in past design) that you can sample from. Not sure if it belongs in doc if were covering just inference, but as far as PyMC goes I think it's a very very nice feature we work hard to include.

Reply:
I would leave this out for now, I guess moving forward we can either further embrace random variable, as currently with the yield it is not immediate obvious how useful to have this concept.